package.json

```diff
@@ -36,6 +36,7 @@
   "dependencies": {
     "moment": "^2.8.4",
     "preact": "^4.5.1",
-    "preact-compat": "^1.7.0"
+    "preact-compat": "^1.7.0",
+    "xterm": "^0.33.0"
```

Comment:
Just for my own understanding, why are there some dependencies in `package.json` and some in `bower.json`? Are these in package.json installed by npm? Are we trying to move away from bower?